lightly/api/api_workflow_upload_embeddings.py

```diff
@@ -14,6 +15,8 @@
 from lightly.utils.io import check_filenames
 
 
+class EmbeddingDoesNotExistError(ValueError):
```

Comment:
Here you define an error for when an embedding does not exist but for the tag you don't - let's try to be consistent

Reply:
I had added one for the embedding to catch this specific error later. Now I added one for the tag.